components/supervisor/pkg/supervisor/tasks.go

```diff
@@ -200,7 +200,7 @@ func (tm *tasksManager) init(ctx context.Context) {
 			presentation.Name = *config.Name
 			title = *config.Name
 		} else {
-			presentation.Name = tm.terminalService.DefaultWorkdir
+			presentation.Name = "Gitpod Task " + strconv.Itoa(i+1)
```

Comment:
I wonder should we also set `title` that it displayed in VS Code like `Gitpod Task 1: npm run watch` instead like now:
<img width="454" alt="Screenshot 2022-04-28 at 16 08 34" src="https://user-images.githubusercontent.com/3082655/165771452-085a6fc3-7daa-4224-aec8-304e9766bd3e.png">

I think it will allow to distinguish task terminals from regular and encourage users to give proper names. wdyt?

Reply:
fyi it is one line more:
```
presentation.Name = "Gitpod Task " + strconv.Itoa(i+1)
title = "Gitpod Task " + strconv.Itoa(i+1)
```

Reply:
Ah! I thought it was somewhere else!

Reply:
✔️ Done!

<img width="1082" alt="image" src="https://user-images.githubusercontent.com/418083/165777660-de503d63-6009-4e10-b607-f67c3e16697f.png">

And `gp tasks lists` output is displayed like this:

<img width="480" alt="image" src="https://user-images.githubusercontent.com/418083/165777894-fcba0a33-ee73-4b48-a5f3-9bb20c09783f.png">